src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
```

Comment:
It's great that you're using constants, but the name `RADIUS` is not descriptive enough to understand what it's used for. Consider renaming it to reflect that it's the default radius for a circle.